README.md

```diff
@@ -27,10 +24,29 @@ The recommended way to install the `enry` command-line tool is to either
 (cd "$(mktemp -d)" && go mod init enry && go get github.com/src-d/enry/v2/cmd/enry)
 ```
 
-Examples
---------
+*enry* CLI accepts similar flags (`--breakdown/--json`) and produce an output, similar to *linguist*:
+
+```bash
+$ enry
+97.71%	Go
+1.60%	C
+0.31%	Shell
+0.22%	Java
+0.07%	Ruby
+0.05%	Makefile
+0.04%	Scala
+0.01%	Gnuplot
+```
+
+Note that enry's CLI **_does not need an actual git repository to work_**, which is intentionally different from the linguist.
```

Comment:
```suggestion
Note that enry's CLI **_does not need an actual git repository to work_**, which is an intentional difference from linguist.
```

Reply:
This is part of the commit under #247 but I can also address it here if that would be more convenient.

Reply:
No, there's fine—I just looked at this PR first. 😄